### PR TITLE
Add Rory the Cat and a setting to change cats

### PR DESCRIPTION
**Pull request by santiagocezar**
Adds Rory and the possibility to add more cats in the future!

image
image

The drawings were made by @AshtakaOOf, and the previous cat is still selectable. 

BTW, what about Rory becoming the default cat?

### CODE REVIEW

**Review by AshtakaOOf — APPROVED** [review]
Cat :+1:

File: launcher/resources/backgrounds/backgrounds.qrc [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
     <qresource prefix="/backgrounds">
-        <file alias="kitteh">catbgrnd2.png</file>
-        <file alias="catmas">catmas.png</file>
-        <file alias="cattiversary">cattiversary.png</file>
+        <file alias="kitteh">kitteh.png</file>
+        <file alias="kitteh-xmas">kitteh-xmas.png</file>
+        <file alias="kitteh-bday">kitteh-bday.png</file>
+        <file alias="rory">rory.png</file>
+        <file alias="rory-xmas">rory-xmas.png</file>
+        <file alias="rory-bday">rory-bday.png</file>
```
Comment (Scrumplex) [inline launcher/resources/backgrounds/backgrounds.qrc:9]: Need to add flat rory

Comment (santiagocezar) [reply launcher/resources/backgrounds/backgrounds.qrc:9]: forgot to add that to the commit, thanks!

**Review by Chew — APPROVED** [review]

**Review by triphora — APPROVED** [review]
Should be merged ASAP

**Review by DioEgizio — APPROVED** [review]

File: launcher/ui/MainWindow.cpp [hunk 2]
```diff
@@ -1560,26 +1560,27 @@ void MainWindow::setCatBackground(bool enabled)
         QDateTime now = QDateTime::currentDateTime();
         QDateTime birthday(QDate(now.date().year(), 11, 30), QTime(0, 0));
         QDateTime xmas(QDate(now.date().year(), 12, 25), QTime(0, 0));
-        QString cat;
+        QString cat = APPLICATION->settings()->get("BackgroundCat").toString();
+
+        qDebug() << "The choosen cat is " << cat << ".";
```
Comment (Scrumplex) [inline launcher/ui/MainWindow.cpp]:
```suggestion
        qDebug() << "The choosen cat is" << cat;
```
Or just remove it altogether maybe

**Review by Scrumplex — APPROVED** [review]

**Review by TheKodeToad — APPROVED** [review]
Looks (trying to avoid a pun) perfect! (apart from the logging)

**Review by Scrumplex — APPROVED** [review]

**Pull request merged**